.github/workflows/nightly.yml

```
@@ -121,7 +121,7 @@ jobs:
     needs: [build, binary, unit-tests]
     strategy:
       matrix:
-        k8s: [1.21.1, 1.20.7, 1.19.11, 1.18.19, 1.17.17, 1.16.15]
+        k8s: [1.21.1, 1.20.7, 1.19.11]
```

```suggestion
        k8s: [1.21.2, 1.20.7, 1.19.11]
```
new minor version of 1.21

---

no 1.22?

---

@lucacome I'll send a separate PR. it will also change the version in the edge version

---

@vepatel yep, no 1.22 for now